app/uk/gov/hmrc/gform/sharedmodel/formtemplate/ComponentType.scala

```diff
@@ -118,6 +118,7 @@ case class Choice(
   options: NonEmptyList[SmartString],
   orientation: Orientation,
   selections: List[Int],
+  hints: Option[NonEmptyList[SmartString]],
```

Comment:
Why done we group options and hints together as a single object and make hint optional if needed? This will avoid the need for validations and looks more practical than keeping them separate. We can always pre-process the json for backwards compatibility no?

Reply:
Yes, option, hint and optionHelpText should be one object ideally, but that is out of scope for GFORMS-790.

Reply:
Ok